### PR TITLE
Nav bar Feature and Front End Login

### PR DESCRIPTION
**Pull request by JimDUc**
Pull requested for code review.

### CODE REVIEW

**Review by JimDUc — COMMENTED** [review]
Pull request viewed. No issues.

**Comment by Jbrooks00** [comment]
Added all group members to our firebase database project. I also implemented firebase, fully functional. I created a base login user interface.

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-<!-- Probably just a placeholder html file for now. Unsure of where to put each individual page as of right now, but we'll figure it out later. -->
+<!DOCTYPE html><!-- Probably just a placeholder html file for now. Unsure of where to put each individual page as of right now, but we'll figure it out later. -->
 
 <app-navbar></app-navbar> <!-- Static Navbar - Anthony -->
 
-<router-outlet></router-outlet> <!-- This actually allows us to determine what buttons display what pages SUPER simply, took me a second to figure out - Anthony -->
\ No newline at end of file
+<div class="content">
+    <router-outlet></router-outlet>
+</div>
```
Comment (avlad83) [inline src/app/app.component.html:7]: the footer component requires another line in this file after this line
<app-footer></app-footer>

File: src/app/app.module.ts [hunk 2]
```diff
@@ -20,7 +32,18 @@ import { NavbarComponent } from './core/navbar/navbar.component';
...
+    provideAuth(() => getAuth()),
+    provideFirestore(() => getFirestore()),
+    AngularFireModule.initializeApp(environment.firebase),
+    BrowserAnimationsModule,
+    MatToolbarModule,
+    MatIconModule,
+    MatButtonModule,
+    MatFormFieldModule,
+    MatInputModule,
+    ReactiveFormsModule
```
Comment (avlad83) [inline src/app/app.module.ts:46]: CoreModule and SharedModule will be necessary for the footer code merge.
Will probably require conflict resolution.

File: src/styles.css [hunk 3]
```diff
@@ -1 +1,32 @@
...
+    justify-content: center;
+}
+
+.margin-top{
+    margin-top: 16px;
+}
+
+.text-center{
+    text-align: center;
+}
```
Comment (avlad83) [inline src/styles.css:32]: This may encounter issues with overwriting Bulma css.

**Review by avlad83 — COMMENTED** [review]
This looks good, may need conflict resolution upon merging.

**Pull request merged**